test/units/nested_fields.js

```diff
@@ -1,36 +1,36 @@
-QUnit.module("Nested fields ( #nested_fields )", {
+QUnit.module('Nested fields ( #nested_fields )', {
   beforeEach: function(){
     $('#nested_fields .additional_information').showWhenThis( {
-      evaluator_callback: function(){ return $('#nested_fields [name="show_additional_information"]').val() == 'show' },
+      evaluator_callback: function(){ return $('#nested_fields [name="show_additional_information"]').val() === 'show' },
```

Comment:
Line is too long.<br>Identifier 'evaluator_callback' is not in camel case.<br>Missing semicolon.